### PR TITLE
Add asl-search to conductor

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: conductor.json [hunk 1]
```diff
@@ -278,6 +301,16 @@
...
       }
+    },
+    {
+      "name": "elasticsearch",
+      "image": "elasticsearch:7.8.0",
+      "port": 9200,
+      "env": {
+        "node.name": "elasticsearch-dev",
+        "discovery.type": "single-node",
+        "ES_JAVA_OPTS": "-Xms512m -Xmx512m"
```
Comment (lennym) [inline conductor.json:312]: WUT?

Comment (wheelsandcogs) [reply conductor.json]: Heap size, I've reduced it - I can't find a minimum amount but I've made it 128mb for now

Comment (wheelsandcogs) [reply conductor.json]: default is 1GB

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "migrate": "docker-compose run asl-schema sh -c 'npm run migrate' && docker-compose exec asl-workflow npm run migrate",
     "seed": "docker-compose run asl-schema sh -c 'npm run seed' && docker-compose exec asl-workflow npm run seed",
     "preseed": "npm run migrate",
+    "postseed": "docker-compose run asl-search sh -c 'npm run indexer'",
```
Comment (lennym) [inline package.json:15]:
```suggestion
    "build-search-index": "docker-compose run asl-search sh -c 'npm run indexer'",
    "postseed": "npm run build-search-index",
```

File: conductor.json (hunk 1 above)
Comment (lennym) [reply conductor.json]: I was about to ask for a comment... but JSON.

😡

**Pull request merged**